applications/DEMApplication/tests/test_DEM_2D.py

```diff
@@ -37,48 +37,27 @@ def GetProblemNameWithPath(self):
         return os.path.join(self.main_path, self.DEM_parameters["problem_name"].GetString())
 
     def FinalizeTimeStep(self, time):
-        tolerance = 1e-3
+        tolerance = 1e-1
```

Comment:
Why such a low tolerance? We won't be able to detect small changes if we increase tolerances.

Reply:
reverted tolerance to its previously value